### DISCUSSION TITLE
beginner question: landing page not rendered as expected on netlify

### DISCUSSION THREAD

**Discussion by anest-uk** [thuliteio/doks#280] [Q&A] [answered]
I have used Hugo and template Learn ok for over a year without problems, so I'm not a total beginner.  However the beautiful doks theme uses node.js and I am not trained on this, maybe that's the problem.

I can do all the steps in the docs to serve doks locally, I can edit etc.  I can deploy to netlify, no errors shown.  Other themes such as Xmin deploy fine for me, that's why I'm asking here in doks.

The issue: the landing page does not get 'styled' (link below).  It seems to lack a stylesheet or is it the 'build' command - netlify default this to `npm run build` so I also tried `hugo`, to no avail.

I have reverted my edits to the unedited theme and the same problem occurs.  The only thing I change is to delete .gitignore, readme and license (as instructed) before push.

This must be so basic!  Help is greatly appreciated!

https://github.com/anest-uk/fornetlify/
https://boring-pike-a6a66c.netlify.app/

**Comment by vidokun** [comment, marked as answer]
have you changed baseurl? I experienced something similar, and succeeded when changing baseurl in the following file

![image](https://user-images.githubusercontent.com/23431072/117559341-a12fe200-b0ae-11eb-9821-73e03a1bc0d1.png)

**Reply by anest-uk** [reply]
Thank you so much that has solved my issue!  

I had customised the _default/config.toml parameter but not the other two.  This issue even pops up in the youtube tutorial I watched, and the tutor records a section on the fix, here paused at the relevant part: https://youtu.be/9Jqvaoeh1W4?t=1437.   But he does not mention the other two config.toml in /next and /production. These need also to be customised.  So his solution is incomplete.

I would have been very stuck without your help, so many thanks.  I wonder if others encounter this issue?

**Reply by vidokun** [reply]
You're welcome 😁

I found similar problem here :
https://github.com/h-enk/doks/discussions/265
